__tests__/app.test.js

```diff
@@ -65,8 +65,35 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
   test("Respond 404: Not Found, when given a non-existent article_id", () => {
-    return request(app).get("/api/articles/33333").expect(404).then(({body}) => {
-        expect(body.msg).toBe('Not Found')
-    })
+    return request(app)
+      .get("/api/articles/33333")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("Respond with 200 and array of articles objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBeGreaterThanOrEqual(1);
+         expect(body.articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
As this is testing a different behaviour, it should be separated into its own test